src/eredis_client.erl

```diff
@@ -463,6 +469,9 @@ do_sync_command(Socket, Command) ->
                 {ok, <<"+OK\r\n">>} ->
                     set_transport_opts(Socket, [{active, once}]),
                     ok;
+                %% {ok, <<"-WRONGPASS invalid username-password pair or user is disabled.\r\n">>}
```

Comment:
there are two such handlings in this module.
two places returning `{error, {unexpected_data, Other}}`, maybe change both ?

Reply:
Good catch, Done